onnx/checker.cc

```diff
@@ -130,12 +137,15 @@ void check_tensor(const TensorProto& tensor, const CheckerContext& ctx) {
     for (const StringStringEntryProto& entry : tensor.external_data()) {
       if (entry.has_key() && entry.has_value() && entry.key() == "location") {
         has_location = true;
-        if (!std::ifstream(ctx.get_model_dir() + entry.value())) {
+        std::string data_path = path_join(ctx.get_model_dir(), entry.value());
+        // use stat to check whether the file exists
+        struct stat buffer;
+        if (stat((data_path).c_str(), &buffer) < 0) {
```

Comment:
!= 0 is a better check
also does this work properly on windows?

Reply:
Yes it works good with Windows.